docs/specs/servers/blockchain/blockchain-permissions-api.md

```diff
@@ -0,0 +1,176 @@
+# Blockchain API Sessions and Permissions
+
+This API is **unstable**, not yet production ready and can be changed at any time.
```

Comment:
I don't think we need to put this disclaimer in specs

Reply:
Makes sense. Removed.